tests/src/common/WhiskProperties.java

```diff
@@ -235,6 +235,10 @@ public static int getControllerPort() {
         return Integer.parseInt(whiskProperties.getProperty("controller.host.port"));
     }
 
+    public static int getMaxActionInvokesPerMinute() {
+        return Integer.parseInt(whiskProperties.getProperty("limits.actions.invokes.perMinute"));
```

Comment:
This may not work for all deployments.
See: https://github.com/openwhisk/openwhisk/blob/master/tests/src/limits/ThrottleTests.scala#L64